README.md

```
@@ -213,8 +213,33 @@ calling. Without it there is no authentication at all, which is why the default
 
+`--addr` and `--port` are independent: `--addr 0.0.0.0` reaches other machines
+on the default port, `--port 8080` moves the port and leaves the binding alone.
+
+### Behind a reverse proxy
+
 The server does not terminate TLS, so anything beyond one machine wants a
-reverse proxy in front of it.
+reverse proxy — Apache, nginx or another — in front of it:
+
+```console
+$ awb serve --public-url https://example.com/awb/ --https
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)